src/structures/Guild.js

```diff
@@ -959,6 +961,52 @@ class Guild extends Base {
       .then(data => GuildAuditLogs.build(this, data));
   }
 
+  /**
+   * Data for a field in Membership Screening
+   * @typedef {Object} GuildMembershipScreeningField
+   * @property {MembershipScreeningType} fieldType The type of the field
+   * @property {string} label The title of the field
+   * @property {string[]} [values] The list of values in the field
+   * @property {boolean} required Whether the user has to fill out this field
+   */
+
+  /**
+   * Data for the Guild Membership Screening object
+   * @typedef {Object} GuildMembershipScreening
+   * @property {boolean} enabled Whether membership screening is enabled
+   * @property {string} description The server description shown in the membership screening form
+   * @property {GuildMembershipScreeningField[]} formFields The steps in the membership screening form
+   */
+
+  /**
+   * Fetches the guild Membership Screening data.
+   * @returns {Promise<GuildMembershipScreening>}
+   * @example
+   * // Fetches the guild membership screening options
+   * guild.fetchMembershipScreening()
+   *   .then(memberScreen => {
+   *     const requiredFields = memberScreen.formFields.filter(field => field.required);
+   *     console.log(`Screening has ${memberScreen.formFields.length} steps, ${requiredFields.length} are required`);
+   *   })
+   *   .catch(console.error);
+   */
+  async fetchMembershipScreening() {
+    if (!this.features.includes('COMMUNITY')) {
+      throw new Error('COMMUNITY');
+    }
+    const data = await this.client.api.guilds(this.id)['member-verification'].get();
```

Comment:
This may be a bug with how the API is currently handling this, see https://github.com/discord/discord-api-docs/issues/2530
A check is still a good idea though, will add.